main.js

```diff
@@ -12,12 +11,13 @@ const shipwrightScript = ({
   const transformedTokenPath = `${outputFolder}/style-dictionary/tokens.json`;
 
   // transform figma token file to style-dictionary ready JSON
-  exec(`node token-transformer ${figmaTokenFile} ${transformedTokenPath}`);
+  execSync(`npx token-transformer ${figmaTokenFile} ${transformedTokenPath}`);
```

Comment:
`token-transformer` was failing silently. Running `node token-transformer` in console produced an error `token-transformer not found` even with packages installed. Changed `exec` to `execSync` to stop race condition - `build` was executing before `token-transformer` had completed.